src/m365/spo/commands/user/user-remove.ts

```diff
@@ -92,13 +137,64 @@ class SpoUserRemoveCommand extends SpoCommand {
     if (this.verbose) {
       await logger.logToStderr(`Removing user from  subsite ${options.webUrl} ...`);
     }
+    try {
+      let requestUrl: string = `${encodeURI(options.webUrl)}/_api/web/siteusers/`;
+      if (options.id) {
+        requestUrl += `removebyid(${options.id})`;
+      }
+      else if (options.loginName) {
+        requestUrl += `removeByLoginName('${formatting.encodeQueryParameter(options.loginName as string)}')`;
+      }
+      else if (options.email || options.userName) {
+        const user = await this.getUser(options.webUrl, options);
+
+        if (user?.Title) {
+          if (this.verbose) {
+            await logger.logToStderr(`Removing user ${user.Title} ...`);
+          }
+          requestUrl += `removebyid(${user.Id})`;
+        }
+        else {
+          throw new Error(`User not found: ${options.userName}`);
+        }
```

Comment:
lets reorder this a bit to something like

```suggestion
        const user = await this.getUser(options.webUrl, options);
        if (!user) {
          throw new Error(`User not found: ${options.userName}`);
        }
        
        if (this.verbose) {
          await logger.logToStderr(`Removing user ${user.Title} ...`);
        }
        requestUrl += `removebyid(${user.Id})`;
```

logically it should be the same think but we avoid one `if/else` block and increase code readabilty